CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v4.2.1 (Upcoming)
```

Comment:
I think this has been bikeshedded enough. Let's please just call it `v5`, and :shipit: